straxen/common.py

```diff
@@ -188,31 +191,12 @@ def get_secret(x):
     message = (f"Secret {x} requested, but there is no environment "
                f"variable {env_name}, ")
     try:
-        from . import xenon_secrets
-    except ImportError:
-        message += ("nor was there a valid xenon_secrets.py "
-                    "included with your straxen installation, ")
-
-        # If on midway, try loading a standard secrets file instead
-        if 'rcc' in socket.getfqdn():
-            path_to_secrets = '/project2/lgrandi/xenonnt/xenon_secrets.py'
-            if os.path.exists(path_to_secrets):
-                sys.path.append(osp.dirname(path_to_secrets))
-                import xenon_secrets
-                sys.path.pop()
-            else:
-                raise ValueError(
-                    message + ' nor could we load the secrets module from '
-                              f'{path_to_secrets}, even though you seem '
-                              'to be on the midway analysis hub.')
-
-        else:
-            raise ValueError(
-                message + 'nor are you on the midway analysis hub.')
+        config = utilix.config.Config()
```

Comment:
I don't like that this would throw out the mechanism that was in place before without any deprecation warning or so. Let's push for using utilix but phase out the xenon_secrets, i.e. throw a deprecation warning such that users should be well aware that they should start using utilix and the corresponding file.

We will then phase out the xenon-secrets.

Reply:
Oh are users using their own xenon secrets file? I thought it was just the single global one? If it's just replacing the global secrets file with a different global file then I don't see the need for a warning, but sure will do.

Reply:
It has to be in your straxen/xenon_sectres.py file. People can have different ones, especially when not running on dali.

Reply:
Ok I see, then yeah a phase out period makes sense.

Reply:
This should be addressed now